apps/src/templates/SchoolInfoInputs.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component, PropTypes } from 'react';
+import CountryAutocompleteDropdown from '@cdo/apps/templates/CountryAutocompleteDropdown';
+import SchoolTypeDropdown from '@cdo/apps/templates/SchoolTypeDropdown';
+import SchoolAutocompleteDropdownWithLabel from '@cdo/apps/templates/census2017/SchoolAutocompleteDropdownWithLabel';
+import SchoolNotFound from '@cdo/apps/templates/SchoolNotFound';
+import i18n from "@cdo/locale";
+
+const SCHOOL_TYPES_HAVING_NCES_SEARCH = ['charter', 'private', 'public'];
+
+const SCHOOL_TYPES_HAVING_NAMES = [
+  'charter',
+  'private',
+  'public',
+  'afterschool',
+  'organization',
+];
+
+export default class SchoolInfoInputs extends Component {
+  static propTypes = {
+    onCountryChange: PropTypes.func.isRequired,
+    onSchoolTypeChange: PropTypes.func.isRequired,
+    onSchoolChange: PropTypes.func.isRequired,
+    onSchoolNotFoundChange: PropTypes.func.isRequired,
+    country: PropTypes.string,
+    schoolType: PropTypes.string,
+    ncesSchoolId: PropTypes.string,
+    schoolName: PropTypes.string,
+    schoolCity: PropTypes.string,
+    schoolState: PropTypes.string,
+    schoolZip: PropTypes.string,
+    useGoogleLocationSearch: PropTypes.bool,
+    fieldNames: PropTypes.object,
+  };
+
+  static defaultProps = {
+    schoolType: "",
+    country: "",
+    ncesSchoolId: "",
+    schoolName: "",
+    schoolCity: "",
+    schoolState: "",
+    schoolZip: "",
+    useGoogleLocationSearch: false,
+    fieldNames: {
+      schoolType: "user[school_info_attributes][school_type]",
+      country: "user[school_info_attributes][country]",
+      ncesSchoolId: "user[school_info_attributes][school_id]",
+      schoolName: "user[school_info_attributes][school_name]",
+      schoolState: "user[school_info_attributes][school_state]",
+      schoolZip: "user[school_info_attributes][school_zip]",
+      googleLocation: "user[school_info_attributes][full_address]",
+    },
+  };
+
+  render() {
+    const isUS = this.props.country === 'United States';
+    const outsideUS = !isUS;
```

Comment:
Is defining outsideUs just so the code is more human readable?

Reply:
Yes, it is for readability. This code was originally from [signup.js](https://github.com/code-dot-org/code-dot-org/blob/f8a1084122f17f1c46c902e0e77aa37f8b279972/apps/src/sites/studio/pages/signup.js#L365) and I kept it as it was.